### ISSUE TITLE
Pokémon: Let's Go, Eevee/Pikachu! – FMV crash with Multicore CPU Emulation

### ISSUE THREAD

**Issue by Dragios** [yuzu-emu/yuzu#5019]
It crash instantly at the start of FMV playback. Sometimes *OpenGL* API will continue playback but exhibits video block corruption instead if it did not crash (as shown at screenshot below).

![Let's Go, Eevee!](https://user-images.githubusercontent.com/18496868/100399589-538a6480-308e-11eb-9eff-6ffda25a01da.png)

Crash log:
```
Debug.Emulated <Critical> core\hle\kernel\svc.cpp:Break:613: Signalling debugger, PANIC! info1=0x0000000000000000, info2=0x0000000000000004
Debug.Emulated <Critical> core\hle\kernel\svc.cpp:Break:665: Emulated program broke execution! reason=0x0000000000000000, info1=0x0000000000000000, info2=0x0000000000000004
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:266: Backtrace, sp=0000000000000000, pc=000000000B4E08A0
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:268: Module Name         Address             Original Address    Offset              Symbol
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:269: 
Render.Vulkan <Info> video_core\renderer_vulkan\vk_pipeline_cache.cpp:GetGraphicsPipeline:216: Compile 0x49F1573A68219740
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: sdk                 000000710004DD0C    000000000B454D0C    000000000004DD0C    _ZN2nn4diag6detail5AbortEPKNS_6ResultE
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: sdk                 000000710004D820    000000000B454820    000000000004D820    _ZN2nn6detail21UnexpectedDefaultImplEPKcS2_i
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: main                00000071005EA308    00000000085EE308    00000000005EA308    
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: main                0000007100029EC4    000000000802DEC4    0000000000029EC4    
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: main                000000710002AB54    000000000802EB54    000000000002AB54    
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: main                0000007100023288    0000000008027288    0000000000023288    
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: main                0000007100020F74    0000000008024F74    0000000000020F74    
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: main                0000007100022D44    0000000008026D44    0000000000022D44    
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: sdk                 00000071000AF8F4    000000000B4B68F4    00000000000AF8F4    
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: sdk                 00000071000B3C54    000000000B4BAC54    00000000000B3C54    
Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: sdk                 00000070F4BF8FFC    FFFFFFFFFFFFFFFC    FFFFFFFFF4BF8FFC    
```
**Version used:** _yuzu 449 | HEAD-mainline-636-3870-g9388ab627_

*System specs:*
**OS:** _Windows 10 Pro Insider Preview build 20262_
**Processor:** _Intel® Core™ i5-2500 @ 3.30GHz_
**RAM:** _8GB DDR3_

**GPU:** _NVIDIA GeForce GTX 1050Ti 4GB GDDR5_
**Driver version:** _460.20_
**Vulkan version:** _1.2.149_

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/5614716/yuzu_log.txt)

**Comment by FearlessTobi** [comment]
Could you please also include the full log file?

**Comment by Dragios** [comment]
Seems like single core emulation works fine with minor video corruption. Multicore is glitched.

**Comment by ameerj** [comment]
fixed by #5799
